docs/src/tut/pos/index.md

```diff
@@ -0,0 +1,243 @@
+# {#point-of-sale} Point-of-sale
+
+This tutorial will walk you through creating a business point-of-sale machine, like Toast or Square.
+
+It assumes that you are comfortable with writing simple Reach DApps and have already completed one or two of our [tutorials](##tuts). 
+
+Complete them all to become a Reach rockstar.
+
+## Program Design
+The main function of our DApp will be to process sale transactions. This means we will need to implement functions for purchases and refunds. 
+
+We will allow purchases of varying amounts for different products, so we will also need to track the amount of each users purchase.
+
+It would also be good to reward our business' visitors with a Loyalty Token. Future programs could use this Loyalty Token to gate access to special features or events.
+
+When designing our DApp we need to consider possibilities for the length of our contract. 
+
+Given the Loyalty Token, it will be best for our implementation to make contract functions accessible until we run out of Loyalty Tokens.
+
+### Here is the general flow of our DApp:
+1. Admin gives parameters and deploys contract.
+2. Admin deposits non-network Loyalty Tokens into contract.
+3. Contract exposes API functions.
+4. Users purchase items in our store.
+5. Users are offered a refund window.
+6. Close access to API functions when Loyalty Token balance is zero.
+7. Transfer network tokens to Admin.
+8. Terminate contract.
+
+## Where the users at?
+At this point of designing our Reach DApp, we need to ask ourselves how many and what types of users will be interacting with it.
+
+We know that we need at least one `{!rsh} Participant` to deploy the contract and serve as our `Admin`. We can consider them our shop owner.
+
+:::note
+All Reach DApps require at least one `{!rsh} Participant`, because `{!rsh} API` members *do not* have the ability to deploy contracts.
+:::
+
+The rest of our users will be customers in our shop, so they are best represented as `{!rsh} API`. This will allow an unlimited number of unknown users to interact with our contract `{!rsh} API` member functions.
+
+Let's write some Reach code for our users.
+```
+load: /examples/point-of-sale/index.rsh
+md5:  2d9202924ee606ab47726038fbde05a3
+range: 1-12
+```
+- Line 4 is a new feature with Reach 0.1.13, read more about `{!rsh} ALGOExitMode`.
```

Comment:
```suggestion
- Line 4 is a new feature with Reach 0.1.13; we need it because we're using boxes, you can read more about it by clicking `{!rsh} ALGOExitMode`.
```